### ISSUE TITLE
How-to to integrate Ionic Framework ?

### ISSUE THREAD

**Issue by samfrach** [preboot/angular-webpack#117]
Hello,

congrats for the environment !

I was wondering what would be the best way to include ionic framework with webpack ? any experience ?

thank you

**Comment by Foxandxss** [comment]
Ionic 2 has webpack already.

**Comment by samfrach** [comment]
How would you do the following : 

- I have an app with this project running for classic browsers
- I would like to use ionic2 components to make my app run with native components
- I want to share my services, pipes and so on

should I have 2 workspaces ? One for classic mode and one for ionic ?

thank you

**Comment by gravity-addiction** [comment]
my ionic2 / angular2 integration only works with rc.4 at the moment. I added an app/ folder to my angular2-webpack folder. Currently I have global services/pipes/directives and two sets of views (web and mobile). 

I learned that all data manipulations should only happen within my services and the views are just for displaying. 

I keep meaning to put up a git repo, waiting for a stable ng2 release and ionic-angular npm module to catch up.
